webpack.common.js

```diff
@@ -0,0 +1,39 @@
+const path = require("path");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
```

Comment:
Probably want to remove this reference

<img width="1105" alt="screen shot 2018-11-29 at 4 38 41 pm" src="https://user-images.githubusercontent.com/5942899/49253718-47af2d80-f3f5-11e8-8d26-331c811d16ac.png">

Reply:
Nice catch - updated